### PR TITLE
feature 1 update

### PR DESCRIPTION
**Pull request by beansoft**

### CODE REVIEW

File: feature1.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Feature 1
 
-update
\ No newline at end of file
+update
```
Comment (beansoft) [inline feature1.md:3]: good

Comment (beansoft) [reply feature1.md:3]: OK

**Review by beansoft — COMMENTED** [review]
please approve

**Comment by JackyJetBrains** [comment]
还有补充吗？

**Pull request merged**